RecoCTPPS/ProtonReconstruction/python/ctppsProtons_cff.py

```diff
@@ -1,7 +1,47 @@
 import FWCore.ParameterSet.Config as cms
 
-from RecoCTPPS.ProtonReconstruction.ctppsProtons_cfi import *
+# import default alignment settings
+from CalibPPS.ESProducers.ctppsAlignment_cff import *
 
-# TODO: remove these lines once conditions data are available in DB
+# import default optics settings
 from CalibPPS.ESProducers.ctppsOpticalFunctions_cff import *
+
+# import and adjust proton-reconstructions settings
+from RecoCTPPS.ProtonReconstruction.ctppsProtons_cfi import *
 ctppsProtons.lhcInfoLabel = ctppsLHCInfoLabel
+
+from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
+from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
+from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
+
+def ApplyDefaultSettings(ctppsProtons):
+  ctppsProtons.association_cuts_45.x_cut_apply = False
+  ctppsProtons.association_cuts_45.y_cut_apply = False
+  ctppsProtons.association_cuts_45.xi_cut_apply = True
+  ctppsProtons.association_cuts_45.xi_cut_value = 0.010
+  ctppsProtons.association_cuts_45.th_y_cut_apply = False
+
+  ctppsProtons.association_cuts_56.x_cut_apply = False
+  ctppsProtons.association_cuts_56.y_cut_apply = False
+  ctppsProtons.association_cuts_56.xi_cut_apply = True
+  ctppsProtons.association_cuts_56.xi_cut_value = 0.015
+  ctppsProtons.association_cuts_56.th_y_cut_apply = False
+
+ctpps_2016.toModify(ctppsProtons, ApplyDefaultSettings) # applied for all Run2 years (2016, 2017 and 2018)
+
+def Apply2017Settings(ctppsProtons):
+  ctppsProtons.association_cuts_45.xi_cut_value = 0.010
```

Comment:
Both these settings are identical to the "default" ones, i.e. you are not replacing anything here

Reply:
Right, in the POG meeting we converged on identical settings for 2016 and 2017. For symmetry (and possible future updates) I kept the code separated for 2016 and 2017.